### PR TITLE
[12.0][MIG] web_listview_invert_selection

### PR DESCRIPTION
**Pull request by Martronic-SA**
pin version,
changed less to scss

### CODE REVIEW

File: web_listview_invert_selection/static/src/js/web_listview_invert_selection.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    ListRenderer.include(/** @lends instance.web.ListView# */{
+
+        events: _.extend({}, ListRenderer.prototype.events, {
+            'click .o_invert_selection': '_onInvertSelection',
+        }),
+
+        _onInvertSelection: function (event) {
+            event.stopPropagation();
+            var checked = self.$("tbody .o_list_record_selector input:checked");
```
Comment (tarteo) [inline web_listview_invert_selection/static/src/js/web_listview_invert_selection.js]: `self` doesn't seem to be defined. Is it still needed? The functionality works in Runbot :confused: :smile:

**Review by tarteo — CHANGES_REQUESTED** [review]

File: web_listview_invert_selection/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (daramousk) [inline web_listview_invert_selection/__manifest__.py]: Remove encoding statements from all the `*.py ` files.

**Review by daramousk — APPROVED** [review]
Some comments

**Comment by Martronic-SA** [comment]
What is the status here? (@espo-tony )

**Pull request closed without merging**